xonsh/ast.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, parser):
         self.lines = None
         self.mode = None
 
-    def ctxvisit(self, node, input, ctx, mode='exec'):
+    def ctxvisit(self, node, input_string, ctx, mode='exec'):
```

Comment:
would you mind changing this to `inp`, rather than `input_string`?  Not a big deal either way...